src/20_Components/amp-sidebar.html

```diff
@@ -91,5 +101,110 @@
     -->
     <button on="tap:sidebar.close" class="ampstart-btn caps m2">Close sidebar</button>
 
+    <p class="previewOnly m2">
+      Please resize the browser window to view the responsiveness for amp-sidebar's toolbar.
+    </p>
+
+  <!-- ## Toolbar -->
+  <!--
+    `toolbar` elements have their own requirments:
+
+    The sidebar may implement toolbars by adding nav elements with the `toolbar` attribute and `toolbar-target` attribute.
+
+    The nav element must be a child of `<amp-sidebar>` and follow this format: `<nav toolbar="(media-query)" toolbar-target="elementID">`.
+
+    The nav containing the toolbar attribute must only contain a single `<ul>` element, that contains `<li>` elements.
+
+    The `<li>` elements may contain any valid HTML elements (supported by AMP), or any of the AMP elements that `<amp-sidebar>` supports.
+
+    Toolbar behavior is only applied while the `toolbar` attribute media-query is valid. Also, an element with the `toolbar-target` attribute id must exist on the page for the toolbar to be applied.
+  -->
+    <div>
+      <amp-sidebar id="sidebar-left" layout="nodisplay" side="left">
+        <amp-img class="amp-close-image" src="/img/ic_close_black_18dp_2x.png" width="20" height="20" alt="close sidebar" on="tap:sidebar-left.close" role="button" tabindex="0"></amp-img>
+        <nav toolbar="(max-width: 56rem)" toolbar-target="target-element-left">
```

Comment:
The reason why I chose to use rem. As it is what amp-start uses currently. So it can cause some weird behavior if toolbar is not changing along with the amp-start elements. 

I'll go ahead, and convert this to px however, and see how it goes 😄